src/module/server-instance/view/server-instance.html

```diff
@@ -0,0 +1,163 @@
+<div class="container-fluid">
+  <div class="row header-row">
+    <div class="col-sm-12 col-md-12">
+      <ol class="breadcrumb">
+        <li>
+          <a href="#/clusters-view-physical" translate>CLUSTERS</a>
+        </li>
+        <li><a ui-sref="server-group({serverGroup: ctrl.$stateParams.serverGroupName})" ui-sref-opts="{reload:true}">{{ctrl.$stateParams.serverGroupName}}</a></li>
+        <li class="active"><strong>{{ctrl.$stateParams.server}}</strong></li>
```

Comment:
Replace with ctrl.serverGroup.name and ctrl.serverInstance.address.name (or create explicit address/name field, might make the template cleaner)